waku/waku_core/time.nim

```diff
@@ -33,3 +33,16 @@ template nanosecondTime*(collector: Gauge, body: untyped) =
     metrics.set(collector, nowInUnixFloat() - start)
   else:
     body
+
+# Unused yet
+#[ proc timestampInSeconds*(time: Timestamp): Timestamp =
+  let timeStr = $time
+  var timestamp: Timestamp = time
+
+  if timeStr.len() > 16:
+    timestamp = Timestamp(time div Timestamp(1_000_000_000))
+  elif timeStr.len() < 16 and timeStr.len() > 13:
+    timestamp = Timestamp(time div Timestamp(1_000_000))
+  elif timeStr.len() > 10:
+    timestamp = Timestamp(time div Timestamp(1000))
+  return timestamp ]#
```

Comment:
This will eventually be used? or we can get rid of it?

Reply:
It's planned to be used. So we keep.